src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -92,16 +92,14 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 var block = (BlockSyntax)localDeclarationStatement.Parent;
                 var declarationIndex = block.Statements.IndexOf(localDeclarationStatement);
 
-                // Comments on the local declaration will move to the next statement.  Format it
-                // so that it the comments show up properly.
+                // Trivia on the local declaration will move to the next statement.
+                // use the callback form as the next statement may be hte place where we're 
```

Comment:
Also, it looks like it's not it "may" be the place - it's always the place isn't it?